src/navigation/tabs/Settings.js

```diff
@@ -1,15 +1,28 @@
 // Vendor
-import React from "react";
-import { StyleSheet, View } from "react-native";
-
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: "center",
-    alignItems: "center",
-    backgroundColor: "#F5FCFF"
+import { createStackNavigator } from "react-navigation";
+
+// Redadalertas
+import SettingsPage from "screens/Settings/SettingsPage";
+import ChangePassword from "screens/Settings/ChangePassword";
+
+const Settings = createStackNavigator(
+  {
+    SettingsPage,
+    ChangePassword
+  },
+  {
+    initialRouteName: "SettingsPage"
+  }
+);
+
+const SettingsTab = createStackNavigator(
+  {
+    Settings
+  },
+  {
+    mode: "modal",
+    headerMode: "none"
```

Comment:
I think we should actually leave these two options off. If we do, then we get to leverage an automatic back button in the header when we navigate to other Settings pages. And we don't have a modal in the Settings area yet.

Reply:
I tried this and it adds some weird spacing to the header, will try to fix later.